pennylane/transforms/rz_phase_gradient.py

```diff
@@ -46,7 +46,7 @@ def _rz_phase_gradient(
 
     precision = len(angle_wires)
     # BasisEmbedding can handle integer inputs, no need to actually translate to binary
-    binary_int = _binary_repr_int(-phi, precision)
```

Comment:
why this change?

Reply:
This is because we change the sign of the phase gradient state in the docs to be `\exp(-2\pi i k/N)` instead of `\exp(2\pi i k/N)`, because the former seems to be used more in the literature. As a consequence, we need to add `\phi` to get a phase of `exp(2\pi i\phi/N)`, which is what we want for `PhaseShift` (there is another relative minus sign between the phases in the resource state and the phase "you get out").